### PR TITLE
Add UDP docs

### PR DESCRIPTION
**Pull request by javierguerragiraldez**

### CODE REVIEW

**Review by dndx — APPROVED** [review]

**Comment by dndx** [comment]
Hello @tlblessing, could you please take a look at this change so it can be ready for the 2.2 release? Thank you!

**Review by kikito — APPROVED** [review]

File: app/2.2.x/plugin-development/custom-logic.md [hunk 1]
```diff
@@ -63,6 +63,11 @@ is a Lua table, and contains values defined by your users, according to your
 plugin's schema (described in the `schema.lua` module). More on plugins schemas
 in the [next chapter]({{page.book.next}}).
 
+Note that UDP streams don't have real "connections".  Kong will consider all
```
Comment (tlblessing) [inline app/2.2.x/plugin-development/custom-logic.md:66]:
```suggestion
Note that UDP streams don't have real connections.  Kong will consider all
```

File: app/2.2.x/plugin-development/custom-logic.md [hunk 2]
```diff
@@ -63,6 +63,11 @@ is a Lua table, and contains values defined by your users, according to your
...
+packets with the same origin and destination host and port as a single
+"connection".  After a configurable time without any packet, the connection is
```
Comment (tlblessing) [inline app/2.2.x/plugin-development/custom-logic.md:68]:
```suggestion
connection.  After a configurable time without any packet, the connection is
```

File: app/2.2.x/plugin-development/custom-logic.md [hunk 3]
```diff
@@ -63,6 +63,11 @@ is a Lua table, and contains values defined by your users, according to your
...
+considered "closed" and the `:log()` function is executed.
```
Comment (tlblessing) [inline app/2.2.x/plugin-development/custom-logic.md:69]:
```suggestion
considered closed and the `:log()` function is executed. 
```

**Review by tlblessing — CHANGES_REQUESTED** [review]
Suggested edits that I'll incorporate. @javierguerragiraldez apologies for the delay. I assume the plugins included in this PR also need a version bump?

**Review by tlblessing — APPROVED** [review]
Approving and merging for 2.2 release but the plugin versions need updating and version files as appropriate for historical updates. We are tracking discrepancies in a spreadsheet.

**Pull request merged**